docs/userguide/basics.md

```diff
@@ -1,17 +1,16 @@
 <!--[metadata]>
 +++
-title = "Get started with containers"
+title = "Quickstart containers"
 description = "Common usage and commands"
 keywords = ["Examples, Usage, basic commands, docker, documentation,  examples"]
 [menu.main]
-parent = "smn_containers"
+parent = "mn_fun_docker"
 +++
 <![end-metadata]-->
 
-# Get started with containers
+# Quickstart containers
```

Comment:
These changes are related to re-structuring the old getting started which was organized around the legacy link feature.